assets/js/atomic/utils/index.js

```diff
@@ -90,27 +65,26 @@ export const DEFAULT_PRODUCT_LIST_LAYOUT = [
 	},
 ];
 
+const getBlockFromComponentName = ( componentName ) => {
+	const block = Object.entries( BLOCK_MAP ).find(
```

Comment:
I'm assuming find is used because the component part cannot be the key of the array in BLOCK_MAP? Just curious. I played around with `reduce` before to index the array how I wanted it to avoid searching, but if it's not possible nevermind :)

Reply:
Yes, I guess creating a reversed object would be more performant than searching it every time. Done in 21fabd7d777e25f7857c8f0fb75095e6f65c01c1.